echopype/tests/convert/test_convert_ek80.py

```diff
@@ -58,23 +58,23 @@ def check_env_xml(echodata):
     assert "sound_velocity_profile_depth"
     assert np.array_equal(echodata["Environment"]["sound_velocity_profile_depth"], [1, 1000])
 
-    # check plat vars
+    # check a subset of plat vars
     plat_vars = {
-        "drop_keel_offset": [np.nan],
+        "drop_keel_offset": [np.nan, 0, 7.5],
         "drop_keel_offset_is_manual": [0, 1],
         "water_level": [0],
         "water_level_draft_is_manual": [0, 1]
```

Comment:
I am puzzled by this block -- aren't these variables all supposed to just a single value? Also in line 73 below, it does not test for actual values, so the fact that the dimensions do not match does not show up.

Reply:
I was very puzzled too, for the same reason as you! What's going on is that `plat_vars` specifies a list of possible *expected* values for each of those variables. The next block, starting in line 68 steps through each of those variables and tests (last line in the block) if the value in the data file matches one of the expected values.

I'll add a comment about this to clarify.

Reply:
I've added the comment. It's worth noting also that the tests for `water_level` and `drop_keel_offset` are not good ones, in general, b/c they assume a limited set of possible values, which in theory is incorrect. They work for the ek80 raw files currently being tested. That's in contrast to the two `*_is_manual` variables, which, based on what I've seen, I assume are boolean and therefore only 0 or 1, so the tests are correct.